src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala

```diff
@@ -142,11 +143,20 @@ private[bigquery] class DirectBigQueryRelation(
 }
 
 object DirectBigQueryRelation {
+  def createHeaderProvider(): HeaderProvider = {
+    FixedHeaderProvider.create("user-agent", BuildInfo.name + "/" + BuildInfo.version)
+  }
```

Comment:
I would inline a one liner only used once in Scala

Reply:
Done.